common/workunit/workunit.cpp

```diff
@@ -3666,7 +3666,7 @@ bool CLocalWorkUnit::archiveWorkUnit(const char *base,bool del,bool ignoredllerr
         return false;
 
     StringBuffer buf;
-    exportWorkUnitToXML(this, buf, false, false);
+    exportWorkUnitToXML(this, buf, false, false, false);
```

Comment:
When rerunning a workunit you would need to resupply the password... I'm not sure that is a bad thing.